site_config/development.js

```diff
@@ -1,163 +1,167 @@
 export default {
-    'en-us': {
-        sidemenu: [
-            {
-                title: 'Contribution Guide',
-                children: [
-                    {
-                        title: 'how to become a committer',
-                        link: '/en-us/docs/development/become a committer.html',
-                    },
```

Comment:
It is recommended not to change these, especially in terms of code

Reply:
I only added a new entry for microbench link as was documented in the README.md file. My linter formatted the file and that's why the whole file shows to be edited.